src/main/java/org/openlmis/stockmanagement/web/stockcardsummariesv2/CanFulfillForMeEntryDto.java

```diff
@@ -50,11 +49,6 @@
   @Setter
   private Integer stockOnHand;
 
-  @Getter
-  @Setter
-  @JsonFormat(shape = STRING)
-  private ZonedDateTime processedDate;
```

Comment:
why this field has been removed? Did you change the response body of an endpoint?

Reply:
I have recovered this field.